iam/sc-enduser-iam.yaml

```diff
@@ -92,23 +92,23 @@ Outputs:
   EndUserGroupArn:
     Value: !GetAtt SCEnduserGroup.Arn
     Export:
-      Name: !Sub '${AWS::Region}-${AWS::StackName}-${RoleAndGroupName}-GroupArn'
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-ServiceCatalogEndusers-GroupArn'
   EndUserGroupName:
     Value: !Ref SCEnduserGroup
     Export:
-      Name: !Sub '${AWS::Region}-${AWS::StackName}-${RoleAndGroupName}-GroupName'
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-ServiceCatalogEndusers-GroupName'
   EndUserRoleId:
     Value: !Ref SCEnduserRole.RoleId
     Export:
-      Name: !Sub '${AWS::Region}-${AWS::StackName}-${RoleAndGroupName}-RoleId'
+      Name: !Sub '${AWS::Region}-${AWS::StackName}-ServiceCatalogEndusers-RoleId'
```

Comment:
couldn't this just be `Name: !Sub '${AWS::Region}-${AWS::StackName}-RoleId`?

`StackName` makes it unique already so it will result in something like `us-east-1-sc-end-users-RoleId`, assuming sceptre we set the stack name to `sc-end-users` on deployment of sc-enduser-iam.yaml

Reply:
I think that would be a nice refinement that could be done in a follow-up PR. I am reluctant to go change this atm -- changing the group names leads to messy build failures because of their use in IAM accounts. I've had it work if you add the new group name in library, then change to the new name in infra, then remove the old name in the library.

Reply:
ok